content/en/docs/concepts/services-networking/dual-stack.md

```diff
@@ -65,12 +65,11 @@ To configure IPv4/IPv6 dual-stack, set dual-stack cluster network assignments:
 * kube-proxy:
   * `--cluster-cidr=<IPv4 CIDR>,<IPv6 CIDR>`
 * kubelet:
-  * when there is no `--cloud-provider` the administrator can pass a comma-separated pair of IP
-    addresses via `--node-ip` to manually configure dual-stack `.status.addresses` for that Node.
-    If a Pod runs on that node in HostNetwork mode, the Pod reports these IP addresses in its
-    `.status.podIPs` field.
-    All `podIPs` in a node match the IP family preference defined by the `.status.addresses`
-    field for that Node.
+  * `--node-ip=<IPv4 IP>,<IPv6 IP>`
+    * This is required for "bare metal" dual-stack nodes (i.e., nodes with no
+      `--cloud-provider`). If you are using a cloud provider, you only need to pass an
```

Comment:
How about:
```suggestion
    * this is required for dual-stack nodes in "bare metal" mode (nodes where you do not specify
      `--cloud-provider` as a command line argument to the kubelet). The kubelet publishes the
      IP addresses that you specify, as `.status.addresses`.  
      If you are using a cloud provider integration, you only need to pass an
```

Does the kubelet ever autodetect dual-stack `.status.addresses`? If so, we should try to tweak the wording further.

Reply:
No; if you specify neither `--cloud-provider` nor `--node-ip`, then kubelet autodetects a _single_ node IP.